dev/tools/make-changelog.sh

```diff
@@ -24,3 +24,6 @@ elif [ "$EDITOR" ]; then
     $EDITOR "$where"
 else echo "$where"
```

Comment:
You can just make this echo unconditional.

Reply:
OK, I just pushed a variant which refine this last line.